### DISCUSSION TITLE
Hardwareeinstellungen Display ESP32

### DISCUSSION THREAD

**Discussion by adminjn** [tbnobody/OpenDTU#1315] [Q&A]
Die neue Softwareversion generic_ESP32<html><body>

  | d90b281
-- | --

</body>
</html>[d90b281](https://github.com/tbnobody/OpenDTU/commits/d90b281)
bringt im Bereich Hardware-Einstellungen Display einen Fehler. Stromsparen aktivieren muss ausgeschaltet sein, um auf dem Display bei Wechselrichterbetrieb etwas zu sehen. Außerdem wird auf dem Display (SSD 1306) Offline mit aktuell richtigen Werten angezeigt.

**Comment by tbnobody** [comment]
Es handelt sich hierbei um keine offzielle Version (diese sind nur unter https://github.com/tbnobody/OpenDTU/releases zu finden) sondern um einen separaten Fork welcher noch nicht integriert ist. Hier kann nur @dAjaY85 was dazu sagen.

**Comment by dAjaY85** [comment]
Diese Änderung beruht auf einen älteren Stand von OpenDTU, da das Thema mit SW-SPI bereits einige Monate alt ist.
